rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/project/ProjectController.java

```diff
@@ -461,6 +465,19 @@ public ResponseEntity searchByExternalIds(@RequestParam MultiValueMap<String, St
         return new ResponseEntity<>(resources, HttpStatus.OK);
     }
 
+    @RequestMapping(value = PROJECTS_URL + "/{id}/attachmentUsage", method = RequestMethod.GET)
+    public @ResponseBody ResponseEntity<Map<String, Object>> getAttachmentUsage(@PathVariable("id") String id)
+            throws TException {
+        User sw360User = restControllerHelper.getSw360UserFromAuthentication();
+        List<AttachmentUsage> attachmentUsages = attachmentService.getAllAttachmentUsage(id);
+        String serializedUsages = attachmentUsages.stream()
+                .map(usage -> wrapTException(() -> THRIFT_JSON_SERIALIZER.toString(usage)))
```

Comment:
Please have a look  at `ThriftJsonSerializer.java`. if it can be useful here.

Reply:
ThriftJsonSerializer.java is part of sw360-portlet module and is not available in resource-server module